ns1/resource_record_test.go

```diff
@@ -127,6 +134,13 @@ func TestAccRecord_meta(t *testing.T) {
 				ImportStateId:     fmt.Sprintf("%s/%s/A", zoneName, domainName),
 				ImportStateVerify: true,
 			},
+			{
+				Config: testAccRecordAnswerMetaDataFeed(rString),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckRecordExists("ns1_record.it", &record),
+					testAccCheckRecordAnswerMetaUp("ns1_datafeed.test", &record),
+				),
+			},
```

Comment:
note that we still aren't testing about diff prevention, just showing that a data feed pointer "works"

Reply:
yeah, I believe there are some TF functions to verify that a diff was/wasnot generated, but this seems fine